src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/TestTemplates/InvalidMergeDiffTemplates/InvalidDiffBlockArray.liquid

```diff
@@ -0,0 +1,5 @@
+{% mergeDiff msg -%}
+[
+  
+]
```

Comment:
How about `{}`?

Reply:
Is valid template, and I added this sample in unit tests.